GliaWidgets/SecureConversations/Confirmation/ConfirmationStyle/ConfirmationStyle+CheckMessagesButtonStyle.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+import UIKit
```

Comment:
UIKit includes Foundation, so I'd like to propose to drop `Foundation`.
```suggestion
import UIKit
```